jazz_test-lambda/components/validate-json.js

```diff
@@ -0,0 +1,38 @@
+
```

Comment:
I don't think this is required. If swagger is updated to take JSON as input, we don't need to validate it again.

Reply:
Awaiting confirmation

Reply:
I see that this util is not being used, can we remove it?

Reply:
done